src/StarcounterClientFiles/wwwroot/sys/app-shell/app-shell.html

```diff
@@ -22,7 +22,9 @@
     <link rel="import" href="/sys/starcounter.html">
     <link rel="import" href="/sys/starcounter-include/starcounter-include.html">
 
-    <link rel="import" href="/sys/bootstrap.html">
+    <link rel="stylesheet" href="/sys/underwear.css/underwear.css">
+    <link rel="stylesheet" href="/sys/uniform.css/uniform.css">
```

Comment:
`uniform.css` does not have to be loaded in the light DOM. It needs to be imported in Shadow DOMs of `starcounter-include`s.